### PR TITLE
Get api topics

### PR DESCRIPTION
**Pull request by SamR232**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+// - 200 OK
+// - 201 Created
+// - 204 No Content
+// - 400 Bad Request
+// - 404 Not Found
+// - 405 Method Not Allowed
+// - 418 I'm a teapot
+// - 422 Unprocessable Entity
+// - 500 Internal Server Error
```
Comment (xmenbteam) [inline __tests__/app.test.js:14]: Nice list, althought we won't be needing 405, 418, 422

File: app.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const app = express();
+const { getTopics } = require("./controllers/topics");
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.all("/*", (req, res) => {
+  res.status(404).send({ message: "Not Found" });
+});
+
```
Comment (xmenbteam) [inline app.js:14]: Think about putting your final 500: server error in here at some point - maybe in a future commit

File: controllers/topics.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const { selectAllTopics } = require("../models/topics");
+
+exports.getTopics = (req, res) => {
+  selectAllTopics().then((userTopics) => {
+    res.status(200).send({ userTopics });
+  });
```
Comment (xmenbteam) [inline controllers/topics.js]: I'd think about renaming this to be honest - your function is called 'getTopics' but you're not sending topics back - you'll be really upset when you're in your FE project and you can't understand why you can't find `topics` on your backend!

**Review by xmenbteam — CHANGES_REQUESTED** [review]
Nice! Just fix those bits and I'll approve it :)

File: __tests__/app.test.js (hunk 1 above)
Comment (SamR232) [reply __tests__/app.test.js:14]: done

File: controllers/topics.js (hunk 3 above)
Comment (SamR232) [reply controllers/topics.js]: I'm not sure what it is sending back, as it's a slug and a description.. is it topic categories?

**Review by xmenbteam — APPROVED** [review]

**Pull request merged**